Kickstarter-iOS/Views/Controllers/CommentRepliesViewControllerTests.swift

```diff
@@ -20,7 +20,8 @@ final class CommentRepliesViewControllerTests: TestCase {
   func testViewController_WithComment_HasRootComment() {
     Language.allLanguages.forEach { language in
```

Comment:
Can update this test too to provide two devices, one iPhone and another iPad.

Reply:
Thanks, issue with copy and paste😄.